projects/flash-cards.md

```diff
@@ -258,8 +258,116 @@ At the beginning of the game, give options for choosing to study different subje
+- The student/team demonstrates misconceptions or confusion around the topic
+- Student/team should prioritize this concept in their studying and practice
+- ✨ An overall "not yet" on one project does **not** mean the module is doomed! We want to see GROWTH. A student/team can struggle with a project and still (taking into account the rest of their work and assessments) demonstrate readiness for the next module! ✨
+
+**Wow** indicates that the student/team did extra work to teach themselves a new concept, or was able to demonstrate a deep & nuanced understanding of a concept  
+- This is not an outcome to prioritize achieving
+- "Wow" should not come at the expense of another concept
+- "Wow" often does not look like extra features, but instead looks like thoughtful refactoring and polish 💅
+</section>
+
+
 ## Evaluation Rubric
 
+This project has 3 evaluated concepts:
+- Functionality
+- JavaScript & Style
+- Test Driven Development
+
+### Minimum Professionalism Expectations
+Note: Professionalism is not a separately graded competency of this project because by now, it is a standard expectation for all your work.  
+
+- Commits are atomic and frequent, effectively documenting the evolution/progression of the application  
+- Developer uses PRs from feature branches before adding new code to the main branch  
+- The README is formatted well and gives new contributors and employers sufficient context about the project:  
+  - Overview of project and goals, technologies used, your code architecture, future features, challenges, wins, and any other reflections  
+  - Setup instructions  
+  - Screenshots or video of your functioning app  
+
+### Functional Expectations
+
+Note: Functionality is important because it is another benchmark to gauge proficiency. However, you should not pursue functionality at the expense of code quality or your learning/growth.  Well-refactored, thoughtful code is better than sloppy extra features.
+
+<section class="answer">
+### Functionality competency examples
+
+- Application fulfills expectations of iterations 1 - 3 with no bugs, crashes, or missing functionality. Game must be playable in the console.
+
+✨WOW✨ can look like:
+
+- Application fulfills expectations of iterations 1 - 4 (iteration 4 means at least one extension) with no bugs, crashes, or missing functionality. Game must be playable in the console.
+</section>
+
+---
+<section class="answer">
+### Fundamental JavaScript & Style competency examples
+
+- Application shows strong effort towards organization, content, and refactoring.  
+- Code leverages JavaScript's [truthy/falsey principles](https://frontend.turing.edu/lessons/module-1/js-truthy-falsy-expressions.html) to write concise conditional logic.  
+- Variable and function naming is meaningful, readable and consistent.  
+- Functions are DRY and demonstrate SRP.  
+- Array prototype methods are used for iteration rather than for loops.  
```

Comment:
`Array prototype methods are used to iterate instead of for loops` 

Just clarifying the sentence a bit!

Reply:
Updated (but not pushed up yet).  Thank you!